contrib/tlp/files/tlp

```diff
@@ -0,0 +1,7 @@
+# tlp service
+
+type            = scripted
+after           = pre-network.target
```

Comment:
what are these `after` supposed to mean? how are they related to power management? especially referencing networkmanager by name seems just wrong

the service is also missing a good hard dependency, which makes it unreliable

Reply:
idk.. it was there in the systemd .service file distributed with tlp...

> missing a good hard dependency

added.

Reply:
I guess it also configures the network interfaces (if available) to be as power-saving as possible...?